bioindex/lib/query.py

```diff
@@ -54,6 +54,29 @@ def fetch_all(config, index, restricted=None, key_limit=None):
     return RecordReader(config, sources, index, restricted=restricted)
 
 
+def fetch_keys(engine, index, columns, restricted=None, key_limit=None):
+    """
+    Construct a SQL query to fetch S3 objects and byte offsets. Run it and
+    return a RecordReader to the results.
+    """
```

Comment:
I don't think this comment is correct here?

Reply:
corrected